packages/amplify-e2e-tests/package.json

```diff
@@ -18,6 +18,7 @@
   "private": true,
   "scripts": {
     "e2e": "npm run setup-profile && jest --verbose",
+    "build-tests": "tsc --build tsconfig.tests.json",
```

Comment:
Because @yuth enabled e2e test compilation as a commit hook as we were ending up non-compiling e2e tests before, which could lead to hard to debug errors when it is just executed with `ts-jest`.